javatests/arcs/core/storage/StorageKeyFactoryTest.kt

```diff
@@ -13,9 +13,27 @@ import org.junit.runners.JUnit4
 @RunWith(JUnit4::class)
 class StorageKeyFactoryTest {
 
+    @Test
+    fun backingStore_namedEntity() {
+        val namedSchema = Schema(setOf(SchemaName("Foo")),
+            SchemaFields(emptyMap(), emptyMap()),
+            "SomeHash"
+        )
+        val arcId = ArcId.newForTest("myId")
+
+        val storeOptionsEmptyName = StorageKeyFactory.BackingStorageKeyOptions(
+            arcId, namedSchema
+        )
+
+        assertThat(storeOptionsEmptyName.unique).isEqualTo(namedSchema.name?.name)
+    }
+
     @Test
     fun backingStore_namelessEntity() {
-        val emptySchema = Schema.EMPTY
+        val emptySchema = Schema(emptySet(),
```

Comment:
nit - i think `emptySet(),` needs to go on a new line as well in kt (here and below)